src/main/java/no/nav/pto/veilarbportefolje/config/ApplicationConfig.java

```diff
@@ -50,13 +46,17 @@ public AzureAdMachineToMachineTokenClient azureAdMachineToMachineTokenClient() {
     }
 
     @Bean
-    public UnleashClient unleashClient(EnvironmentProperties properties, AuthContextHolder authContextHolder) {
-        return new UnleashClientImpl(properties.getUnleashUrl(), APPLICATION_NAME, List.of(new ByUserIdStrategy(authContextHolder)));
-    }
-
-    @Bean
-    public UnleashService unleashService(UnleashClient unleashClient) {
-        return new UnleashService(unleashClient);
+    public DefaultUnleash defaultUnleash(EnvironmentProperties properties) {
+        String appName = "veilarbportefolje";
```

Comment:
Denne er en konstant høyere opp i filen, så kan bare bruke `APPLICATION_NAME` rett i configen i steden for å deklarere en variabel til :)